### PR TITLE
[FIX] floors undefined error

### PR DESCRIPTION
**Pull request by GabbasovDinar**

### CODE REVIEW

File: pos_mobile/static/src/js/gui.js [hunk 1]
```diff
@@ -37,7 +37,7 @@ odoo.define('pos_mobile.gui', function (require) {
             } else if (current_screen === 'payment') {
                 var height = $('.payment-screen .right-content').height();
                 var paymentmethods = $('.payment-screen .paymentmethods-container').height();
-                $('.paymentlines-container').css({height: height - paymentmethods - 600});
+                $('.paymentlines-container').css({height: height - paymentmethods - 750});
```
Comment (yelizariev) [inline pos_mobile/static/src/js/gui.js]: Can this number 760 be computed?

File: pos_mobile_restaurant/static/src/js/chrome.js [hunk 2]
```diff
@@ -45,9 +45,11 @@ odoo.define('pos_mobile_restaurant.chrome', function (require) {
             $(".mobile-floor-selector").append(floor_selector);
 
             // floors
-            this.pos.config.floor_ids.forEach(function(id, index){
-                self.rerender_floors(id, index, true);
-            });
+            if (this.pos.floors) {
+                this.pos.floors.forEach(function(floor, index){
```
Comment (yelizariev) [inline pos_mobile_restaurant/static/src/js/chrome.js]: don't use forEach
wrong

**Review by yelizariev — CHANGES_REQUESTED** [review]

File: pos_mobile/static/src/js/gui.js (hunk 1 above)
Comment (GabbasovDinar) [reply pos_mobile/static/src/js/gui.js]: @yelizariev it's fixed size for numpad

Comment (yelizariev) [reply pos_mobile/static/src/js/gui.js]: Add comment what 750 is (how to compute it) or make it computable

File: pos_mobile_restaurant/static/src/js/floors.js [hunk 3]
```diff
@@ -48,6 +42,28 @@ odoo.define('pos_mobile_restaurant.floors', function (require) {
...
             }
             this._super();
+        },
+        /*
+         The 'drag' evets in original code was added because on touch devices it is sometimes
+         not easy to click, especially on small elements. You have to touch and
+         release the screen without moving your finger.
+
+         But the table selection screen don't hides overflow in mobile version and we cannot to use
+         the scrolling when touch a table.
```
Comment (yelizariev) [inline pos_mobile_restaurant/static/src/js/floors.js]: I don't understand

File: pos_mobile_restaurant/static/src/js/floors.js [hunk 4]
```diff
@@ -48,6 +42,28 @@ odoo.define('pos_mobile_restaurant.floors', function (require) {
...
+
+         TODO: Make without remove events
+         */
+        update_click_handlers: function(editing){
+            this._super(editing);
+            if (!editing) {
+                this.$el.off('dragend');
```
Comment (yelizariev) [inline pos_mobile_restaurant/static/src/js/floors.js]: Put your comment inside ``if`` block

**Review by yelizariev — CHANGES_REQUESTED** [review]

**Review by yelizariev — APPROVED** [review]
code is reviewed

**Review by ilmir-k — APPROVED** [review]
:+1:

**Pull request merged**